src/Documentation/Documentation.v4.0.shfbproj

```diff
@@ -38,7 +38,7 @@
     <HelpTitle>openstack.net API Reference Documentation</HelpTitle>
     <ContentPlacement>AboveNamespaces</ContentPlacement>
     <RootNamespaceTitle>API Reference</RootNamespaceTitle>
-    <WebsiteSdkLinkType>Msdn</WebsiteSdkLinkType>
+    <WebsiteSdkLinkType>None</WebsiteSdkLinkType>
```

Comment:
Here's the [original](http://www.openstacknetsdk.org/docs/html/M_net_openstack_Core_Providers_IObjectStorageProvider_CopyObject.htm) and the [output after the PR](http://104.239.183.134/browse/OSTK-NETPR-9/artifact/shared/Documentation/html/M_net_openstack_Core_Providers_IObjectStorageProvider_CopyObject.htm). You're right it's missing the links back to MSDN for references to framework classes like System.String.  

I was looking for ways to improve build time and this was a suggested change. It saves about a minute. I was on the fence as to how valuable the links really are. Thoughts?